Code/GraphMol/SubstructLibrary/SubstructLibrary.cpp

```diff
@@ -322,6 +380,16 @@ bool SubstructLibrary::hasMatch(const ROMol &query, bool recursionPossible,
              .size() > 0;
 }
 
+bool SubstructLibrary::hasMatch(const TautomerQuery &query,
```

Comment:
This (and the overload above it) could also be template methods defined in the header.